index.html

```diff
@@ -4,27 +4,187 @@
-                <li><a href="https://medium.com/"><img src="./icons/medium.svg"></a></li>
-            </ul>
+        <div class="main-container">
+            <div id="home-screen">
+                <nav id="navbar">
+                    <a href="">Md Tariq-ul Islam</a>
+                    <a href=""><img src="./icons/Menu.svg"></a>
+                </nav>
+                <div id="welcome">          
+                    <h1>Hey There.</br>I'm Tariq-ul Islam</h1>
+                    <h2>I am a Software Developer</h2>
+                    <p>I can help you build a product, feature or website look through some of my work and experience. If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+                    <ul id="social-media">
+                        <li><a href="https://github.com/"><img src="./icons/github.svg"></a></li>
+                        <li><a href="https://www.linkedin.com/"><img src="./icons/Linkedin.svg"></a></li>
+                        <li><a href="https://angel.co/"><img src="./icons/Angelist.svg"></a></li>
+                        <li><a href="https://twitter.com/"><img src="./icons/twitter.svg"></a></li>
+                        <li><a href="https://medium.com/"><img src="./icons/medium.svg"></a></li>
+                    </ul>
+                </div>
+            </div>
+            <main>
+                <div id="work-heading">
+                    <label>My Recent Works</label>
+                </div>
+                <div class="works">
+                    <div class="work snapshot">
+                        <img src="./images/work1.jpg" alt="My First Project">
+                    </div>
+                    <div class="work project-name">
+                        <h2>Multi Post Stories</h2>
+                    </div>
+                    <ul class="work project-summary">
+                        <li>Canopy</li>
+                        <li>Back End Dev</li>
+                        <li>2015</li>
+                    </ul>
+                    <div class="work description">
+                        <p>
+                            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+                        </p>
+                    </div>
+                    <div class="work languages">
+                        <label>CSS</label>
+                        <label>HTML</label>
+                        <label>Bootstrap</label>
+                        <label>Ruby</label>
+                    </div>
+                    <a class="work see-project" href="">See Project</a>
+                </div>
+                <div class="works">
+                    <div class="work snapshot">
+                        <img src="./images/Illustration.svg" alt="My First Project">
```

Comment:
- [ ] Kindly Also note that its a requirement that you use the colors, images from the figma design as much as possible. Your colors  and images are currently differing